hedera-mirror-common/src/test/java/com/hedera/mirror/common/domain/DomainBuilder.java

```diff
@@ -464,6 +465,10 @@ public byte[] create2EvmAddress() {
         return bytes(20);
     }
 
+    public ByteString create2EvmAddressByteString() {
+        return UnsafeByteOperations.unsafeWrap(create2EvmAddress());
```

Comment:
Use the already present logic
```suggestion
        return DomainUtils.fromBytes(create2EvmAddress());
```

Reply:
Thanks, I have removed the method entirely.